spec/filters/unsupported/string.rb

```diff
@@ -1,6 +1,7 @@
 # NOTE: run bin/format-filters after changing this file
 opal_unsupported_filter "String" do
   fails "BasicObject#__id__ returns a different value for two String literals"
+  fails "Module#const_defined? returns true when passed a constant name with EUC-JP characters"
```

Comment:
Is this related? Do you know why we need to ignore these?

Reply:
Those tests pass in an encoding that is not supported by Opal.  This patch restores the Opal 1.0.5 behavior of throwing an ArgumentError when an encoding is provided that is not supported.